doc/rtd/topics/debugging.rst

```diff
@@ -163,3 +163,63 @@ commandline:
 
 Inspect cloud-init.log for output of what operations were performed as a
 result.
+
+.. _proposed_sru_testing:
+
+Manually verifying cloud-init Stable Release Updates (SRU)
+==============================================================
+In order to retain stability on released Ubuntu series, cloud-init performs
+the following SRU procedure for `CloudinitUpdates`_.
+
+To manually test cloud-init **-proposed** versions
+before they are publicly released to **-updates** pockets:
+
+1. Launch a VM on your favorite platform, providing the cloud-config
```

Comment:
As this instruction is written currently, you have to work backwards through it: execute the code snippet below it, then launch the VM. Maybe we should split this into two steps: (1) is "write out srutest.yaml", and (2) is "launch a VM"?

Reply:
I've adopter your other suggestion of just surfacing the yaml that they need to use, leaving an exercise to the reader to figure out how they write their file.

Reply:
Thanks!